src/dmd/dmodule.d

```diff
@@ -658,13 +658,13 @@ extern (C++) final class Module : Package
      */
     bool read(const ref Loc loc)
     {
-        if (srcBuffer)
+        if (this.src)
             return true; // already read
```

Comment:
So one only needs to set `src` if the reading from stdin is already taken care of?  I think this fine.

Reply:
Yeah that's a good point, and would be an argument to leave it in `mars.d` 🤔 
Do you use `readFromStdin` in GDC ?